lunatrace/bsl/hasura/migrations/lunatrace/1656548293311_use_new_vulnerability_table/down.sql

```diff
@@ -0,0 +1 @@
+SELECT NULL;
```

Comment:
We need a down migration here.  Maybe we can all discuss not having down migrations anymore as a team but for now, this breaks my dev ex if its null